helm-charts/common/mm-embedding/README.md

```diff
@@ -0,0 +1,58 @@
+# OPEA mm-embedding microservice
+
+Helm chart for deploying OPEA multimodal embedding service.
+
+## Installing the Chart
+
+To install the chart, run the following:
+
+```console
+cd GenAIInfra/helm-charts/common
+export MODELDIR=/mnt/opea-models
+export HFTOKEN="insert-your-huggingface-token-here"
+# To deploy embedding-multimodal-bridgetower microserice on CPU
+helm install mm-embedding mm-embedding --set global.modelUseHostPath=${MODELDIR} --set global.HUGGINGFACEHUB_API_TOKEN=${HFTOKEN}
+# To deploy embedding-multimodal-bridgetower microserice on Gaudi
```

Comment:
Why only this bridgetower option on Gaudi has been commented? What about the clip option afterwards?

Reply:
forgot to comment out the clip case. Clip only support CPU case now.